v3/otm_meshless.cpp

```diff
@@ -496,12 +497,21 @@ void otm_allocate_state(input const& in, state& s) {
   auto const num_elements = s.elements.size();
   auto const num_materials = in.materials.size();
   auto const num_boundaries = in.boundaries.size();
-  auto const support_size = s.nodes_in_element.size();
   s.u.resize(num_nodes);
   s.v.resize(num_nodes);
   s.V.resize(num_points);
-  s.grad_N.resize(num_points * support_size);
-  s.N.resize(num_points * support_size);
+  if (s.points_to_point_nodes.size() == 0) {
```

Comment:
Have we already read and converted the tet Exodus mesh to a meshless data structure at this point? If so, shouldn't points_to_point_nodes be populated? points_to_point_nodes should be resized so that each of its elements is nodes_in_elements.size(). So even then the algorithm in the "else" branch would give the correct result. It's difficult to see which routine "owns" sizing and filling of points_to_point_nodes from here.